public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -25,4 +27,64 @@
 function load() {
 	require_once PLUGIN_DIR . 'includes/class-feedback.php';
 	require_once PLUGIN_DIR . 'includes/comment.php';
+	require_once PLUGIN_DIR . 'includes/form.php';
+	require_once PLUGIN_DIR . 'includes/page.php';
+}
+
+/**
+ * Create main Feedback page.
+ */
+function activate() {
+	add_feedback_page();
+}
+
+/**
+ * Create the Feedback page, save ID into an option.
+ */
+function add_feedback_page() {
+	$page_id = wp_insert_post( array(
```

Comment:
Since the content is getting overwritten by a filter, I wonder if it would be helpful to insert an admin note into the post content here explaining how this page works.